tests/test_storeoutput.py

```diff
@@ -0,0 +1,145 @@
+import numpy as np
+import pytest
+from aicconnector.storeoutput import get_frame_from_sae_message
+from aicconnector import storeoutput
+from aicconnector.storeoutput import _annotate
+
+class _Frame:
```

Comment:
Why do we create mock objects of actual vision-api types in this test? Protobuf objects have a slightly unusual behavior in setting and getting fields, which this does not replicate.